README.md

```diff
@@ -99,27 +99,36 @@ Here are the default block classes that can be extended:
 
 | Class Name | Noteworthy Behavior |
 | --- | --- |
-| BookmarkBlock | |
+| BookmarkBlock | Converts visual bookmark into plain text link in markdown, using the caption as the link text. |
+| BreadcrumbBlock | Ignore |
 | BulletedListItemBlock | |
 | CalloutBlock | The content of the callout block is extracted, but the emoji and background color are ignored. |
-| ChildPageBlock | |
-| FencedCodeBlock | |
+| ChildDatabaseBlock | Converts database to markdown in different file in directory and writes a relative link to the markdown|
+| ChildPageBlock | Converts Child Page to markdown. |
```

Comment:
Depending on the selected output format Command-line argument, we may or many not be exporting the child pages into markdown. In fact, currently, we don't ever export child page objects. In the future, we will have a recursive export format option that will recursively go through a page and export everything.

Reply:
For now, I propose we just ignore child pages. In the future, I think the solution you have outlined is a good one.